### PR TITLE
 ✨Ensure Smoother F5 Experience

### PR DESCRIPTION
**Pull request by danielmackay**
Add option of both local DB and docker DB

### CODE REVIEW

File: src/WebApi/appsettings.json [hunk 1]
```diff
@@ -5,8 +5,12 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  // NOTE: If any of your connection strings real passwords, you should delete them and put them in the user secrets file instead.
```
Comment (william-liebenberg) [inline src/WebApi/appsettings.json:8]: does json comments actually work as expected? i haven't dared use json comments in a lonnnnnnng time.

Locally vs deployed to an appservice shouldn't matter right?

Comment (danielmackay) [reply src/WebApi/appsettings.json:8]: I'll give this a test.  Thanks for the suggestion.

File: src/WebApi/appsettings.json [hunk 2]
```diff
@@ -5,8 +5,12 @@
...
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=CleanArchitecture;Trusted_Connection=True;MultipleActiveResultSets=true;TrustServerCertificate=True"
+    // Local DB
+    //"DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=CleanArchitecture;Trusted_Connection=True;MultipleActiveResultSets=true;TrustServerCertificate=True"
+    // Docker DB
+    "DefaultConnection": "Server=localhost,1433;Initial Catalog=CleanArchitecture;Persist Security Info=False;User ID=sa;Password=yourStrong(!)Password;MultipleActiveResultSets=True;TrustServerCertificate=True;Connection Timeout=30;"
```
Comment (christoment) [inline src/WebApi/appsettings.json:13]: How about just leave this blank, and put this in the Development.json?

The idea is that local development (might) work, but production will never work without the proper config.

Comment (danielmackay) [reply src/WebApi/appsettings.json:13]: Good idea.

Comment (danielmackay) [reply src/WebApi/appsettings.json]: Done

File: src/WebApi/appsettings.json (hunk 1 above)
Comment (danielmackay) [reply src/WebApi/appsettings.json]: Gave this a go, and everything works fine.

https://deleteme-dev-app.azurewebsites.net/swagger

image

image

**Review by christoment — APPROVED** [review]

**Pull request merged**